backend/app/api/endpoints/internal/rag.py

```
@@ -690,6 +693,58 @@ async def get_all_chunks(
         raise HTTPException(status_code=500, detail=str(e))
 
 
+@router.post("/purge-knowledge-index")
+async def purge_knowledge_index(
+    request: RemotePurgeKnowledgeIndexRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)